presto-main/src/main/java/com/facebook/presto/server/security/LdapBinder.java

```diff
@@ -13,11 +13,9 @@
  */
 package com.facebook.presto.server.security;
 
-import javax.naming.directory.DirContext;
-
 public interface LdapBinder
 {
     String getBindDistinguishedName(String user);
 
-    boolean checkForGroupMembership(String user, String groupDistinguishedName, DirContext context);
+    String groupCheckSearchFilter(String user, String groupDistinguishedName);
```

Comment:
I agree. I would like to go with getGroupSearchFilter
@anusudarsan : Any preferences.